test/component_test.go

```
@@ -0,0 +1,155 @@
+			assert.Equal(t, fmt.Sprintf("%s-lock", bucketID), dynamodbTableID)
+
+			dynamodbTableARN := atm.Output(component, "tfstate_backend_dynamodb_table_arn")
+			assert.True(t, strings.HasSuffix(dynamodbTableARN, dynamodbTableID))
+
+			accessRoleARNs := atm.OutputMapOfObjects(component, "tfstate_backend_access_role_arns")
+			assert.NotEmpty(t, accessRoleARNs)
+
+			policyString := aws.GetS3BucketPolicy(t, awsRegion, bucketID)
+
+			var policy BucketPolicy
+			json.Unmarshal([]byte(policyString), &policy)
```

golangci-lint (1.62.2)

Error return value of `json.Unmarshal` is not checked

(errcheck)